x-pack/plugins/security_solution/common/endpoint/data_generators/host_isolation_exception_generator.ts

```diff
@@ -10,14 +10,17 @@ import { ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID } from '@kbn/securitysolutio
 import { BaseDataGenerator } from './base_data_generator';
 import { getCreateExceptionListItemSchemaMock } from '../../../../lists/common/schemas/request/create_exception_list_item_schema.mock';
 
+const EFFECT_SCOPE_TYPES = ['policy', 'global'];
+
 export class HostIsolationExceptionGenerator extends BaseDataGenerator<CreateExceptionListItemSchema> {
   generate(): CreateExceptionListItemSchema {
+    const scopeType = this.randomChoice(EFFECT_SCOPE_TYPES);
     const overrides: Partial<CreateExceptionListItemSchema> = {
       name: `generator exception ${this.randomString(5)}`,
       list_id: ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID,
       item_id: `generator_endpoint_host_isolation_exception_${this.randomUUID()}`,
       os_types: ['windows', 'linux', 'macos'],
-      tags: ['policy:all'],
+      tags: scopeType === 'global' ? ['policy:all'] : ['policy:'],
```

Comment:
Suggestion: changing `EFFECT_SCOP_TYPES` array above to just store `policy:all` and `policy:` would simplify this here by just `this.randomChoice(EFFECT_SCOPE_TYPES)`

Reply:
great idea. I'll do the change.